core/src/main/java/jenkins/util/AntClassLoader.java

```diff
@@ -918,6 +911,26 @@ public URL getResource(String name) {
         return url;
     }
 
+    /**
+     * Finds a matching file by iterating pathComponents.
+     *
+     * @param pathComponents Path to a folder, split into the individual folder names
+     * @param name File to find
+     * @return Url to found object
+     */
+    protected URL getUrl(Vector pathComponents, String name) {
```

Comment:
Can it be null? Looks so according to the original code

Reply:
Are you suggesting the comment should note this?

Reply:
```suggestion
    @CheckForNull
    protected URL getUrl(Vector pathComponents, String name) {
```

+ maybe import